### PR TITLE
feat Add numofdigits subject

### PR DESCRIPTION
**Pull request by zainabdnaya**

### CODE REVIEW

File: subjects/numofdigits/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+## number of digits
+
+### Instructions
+
+Write a function that returns the number of digits in a nonnegative integer n.
+- if you enter a negative input return 0.
```
Comment (Hamzaelkhatri) [inline subjects/numofdigits/README.md:6]: > if the number is negative input returns 0.

File: subjects/numofdigits/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+	"piscine"
+)
+
+func main() {
+	z01.PrintRune(piscine.Numofdigits(3))
+	z01.PrintRune(piscine.Numofdigits(245))
+	z01.PrintRune(piscine.Numofdigits(-1))
+	z01.PrintRune(piscine.Numofdigits(885))
+	z01.PrintRune(piscine.Numofdigits(8574))
+	z01.PrintRune('\n')
```
Comment (Hamzaelkhatri) [inline subjects/numofdigits/README.md:32]: no need to add this line

File: subjects/numofdigits/README.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+```go
+package main
+
+import (
+	"github.com/01-edu/z01"
+	"piscine"
+)
+
+func main() {
+	z01.PrintRune(piscine.Numofdigits(3))
```
Comment (Hamzaelkhatri) [inline subjects/numofdigits/README.md:27]: use `fmt.Println` or 
```go 
z01.PrintRune(Rune(piscine.Numofdigits(3) + '0'))
```

**Review by Hamzaelkhatri — CHANGES_REQUESTED** [review]

**Review by Hamzaelkhatri — APPROVED** [review]

File: subjects/numofdigits/README.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
+## number of digits
+
+### Instructions
+
+Write a function that returns the number of digits in a nonnegative integer n.
```
Comment (MSilva95) [inline subjects/numofdigits/README.md:5]: Why not use just "positive integer" instead?

File: subjects/numofdigits/README.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+```
+
+### Usage
+
+```go
+package main
+
+import (
+	"fmt"
+	"piscine"
```
Comment (MSilva95) [inline subjects/numofdigits/README.md:23]: Remove Piscine please

**Review by MSilva95 — CHANGES_REQUESTED** [review]

File: subjects/numofdigits/README.md (hunk 5 above)
Comment (zainabdnaya) [reply subjects/numofdigits/README.md]: done

File: subjects/numofdigits/README.md (hunk 4 above)
Comment (zainabdnaya) [reply subjects/numofdigits/README.md]: Done

**Review by MSilva95 — APPROVED** [review]

**Pull request merged**